torch/onnx/utils.py

```diff
@@ -127,8 +127,22 @@ def _optimize_graph(graph, operator_export_type, _disable_torch_constant_prop=Fa
         torch._C._jit_pass_onnx_preprocess_caffe2(graph)
 
         if operator_export_type == OperatorExportTypes.ONNX_ATEN_FALLBACK:
+            torch.onnx.symbolic_helper._quantized_ops.clear()
+            # Unpack quantized weights for conv and linear ops and insert into graph.
             torch._C._jit_pass_onnx_unpack_quantized_weights(graph, params_dict)
 
+            # Insert permutes before and after each conv op to ensure correct order.
+            torch._C._jit_pass_onnx_quantization_insert_permutes(graph, params_dict)
```

Comment:
Is it possible we can use _jit_pass_custom_pattern_based_rewrite_graph to insert permutes as well?

Reply:
I did try that but got an assert of the form 
```
terminate called after throwing an instance of 'c10::Error'
  what():  n->owningGraph() == this && n->inBlockList() INTERNAL ASSERT FAILED at ../torch/csrc/jit/ir.h:1191
```
I suspect adding new ops to the graph isn't updating all the required dependencies correctly.